app/controllers/evaluations_controller.rb

```diff
@@ -86,6 +86,21 @@ def cancel_point_request
                 notice: t(:cancelled_point_request)
   end
 
+  def copy_previous_principles
+    authorize_evaluation
+
+    previous_evaluation = current_group.evaluations.second_to_last
+    previous_evaluation.principles.each do |principle|
+      Principle.create!(evaluation: current_evaluation,
+                        name: principle.name,
+                        description: principle.description,
+                        type: principle.type,
+                        max_per_member: principle.max_per_member)
+    end
+    flash[:notice] = t('evaluation.successful_principle_import')
+    redirect_to group_evaluation_principles_path(current_group, current_evaluation)
+  end
```

Comment:
What do you think about extracting this action to a new one instead of increasing complexity here? I'm thinking about a controller named `evaluations/copy_principles` where the #POST is responsible for this action. Though I'm not entirely content with the naming, it's open for discussion.

Reply:
I aggree, that the `EvaluationsController` is too crowded, and somehow we should make it easier to read.
A CopyPrinciplesController would be weird for me, it could only have one action because the name is to specific. 
Breaking up the evaluation related actions could be hard to follow, I would prefer to move the majority of the logic from the `EvaluationsController` to separate services. This could be a larger refactor so I would create a separate ticket for it.
What do you think?

Reply:
Sounds good to me.